assets/packages/adminbasics/src/components/file-manager.js

```diff
@@ -0,0 +1,201 @@
+const replacingIconData = [
+    {
+        originIcon: "a[href='kcact:upload'] span",
+        newIcon: "ri-upload-fill",
+    },
+    {
+        originIcon: "a[href='kcact:refresh'] span",
+        newIcon: "ri-refresh-line",
+    },
+    {
+        originIcon: "a[href='kcact:settings'] span",
+        newIcon: "ri-settings-3-line",
+    },
+    {
+        originIcon: "a[href='kcact:maximize'] span",
+        newIcon: "ri-fullscreen-line",
+    },
+    {
+        originIcon: "a[href='kcact:about'] span",
+        newIcon: "ri-information-line",
+    },
+    {
+        originIcon: "a[href='kcdir:/files'] span.folder",
+        newIcon: "ri-folder-line",
+    },
+    {
+        originIcon: "a[href='kcdir:/images'] span.folder",
+        newIcon: "ri-folder-line",
+    },
+    {
+        originIcon: "a[href='kcdir:/flash'] span.folder",
+        newIcon: "ri-folder-line",
+    },
+];
+
+const menuReplacingIconData = [
+    {
+        originIcon:
+            "a[href='kcact:refresh'][class='ui-menu-item-wrapper'] span",
+        newIcon: "ri-refresh-line",
+    },
+    {
+        originIcon: "a[href='kcact:download'] span",
+        newIcon: "ri-download-line",
+    },
+    {
+        originIcon: "a[href='kcact:mkdir'] span",
+        newIcon: "ri-folder-add-line",
+    },
+    {
+        originIcon: "a[href='kcact:mvdir'] span",
+        newIcon: "ri-eraser-line",
+    },
+    {
+        originIcon: "a[href='kcact:rmdir'] span",
+        newIcon: "ri-delete-bin-line",
+    },
+
+    // folder regular
+];
+
+const handleReplaceIcons = ({ iframeSource, originIcon, newIcon }) => {
```

Comment:
All these functions are missing docblocks. Is that by design? What's our current policy?